test/formatDate.spec.js

```diff
@@ -0,0 +1,26 @@
+import { expect } from "chai";
+import React from "react";
+import { shallow } from "enzyme";
+import Globalize from "globalize";
+import FormatDate from "../dist/date";
+
+Globalize.load(
+    require( "cldr-data/main/en/ca-gregorian" ),
+    require( "cldr-data/main/en/timeZoneNames" ),
+    require( "cldr-data/main/en/numbers" ),
+    require( "cldr-data/supplemental/likelySubtags" ),
+    require( "cldr-data/supplemental/timeData" ),
+    require( "cldr-data/supplemental/weekData" )
+);
+
+describe("formatDate Component", () => {
+    it("renders as a <span>", () => {
+        const wrapper = shallow(<FormatDate locale="en" pattern="GyMMMd">{new Date()}</FormatDate>);
+        expect(wrapper.type()).to.eql("span");
+    });
+
+    it("formats date as 1/1/2016", () => {
+        const wrapper = shallow(<FormatDate locale="en" pattern="GyMMMd">{new Date("Jan 01 2016")}</FormatDate>);

```

Comment:
The tested value doesn't correspond to the given pattern. The tested value is the default value when no options is passed. It would be really weird if this worked, I was starting to get concerned wondering that react-globalize had developed AI. :smile: :stuck_out_tongue_closed_eyes: See https://github.com/jquery-support/react-globalize/blob/master/src/date.js#L5